opencti-platform/opencti-front/src/private/components/workspaces/WorkspaceCreation.jsx

```diff
@@ -123,6 +127,17 @@ const WorkspaceCreation = ({ paginationOptions, type }) => {
 
   const createDashboardButton = FAB_REPLACED ? (props) => (
     <>
+      <GradientButton
```

Comment:
Personally I find it better to always have this button and no speeddial

Reply:
Hmm. 
Does that mean we're going to change the UI all the time? 
I admit that I don't really know how feature flags are managed.

Reply:
I changed the behavior to always show the button on top of the datatable